docs/Planner.md

```diff
@@ -1,63 +1,86 @@
-# Planner Design
+# Query Planner and Optimizer
 
-This document lays out the design of the planner.
+This document describes the design of the Flux query planner and optimizer.
 
-## Interface
+## Plans
+--------
 
-The Planner inter is defined as:
+A Flux query plan is represented as a Directed Acyclic Graph (DAG).
+The nodes of the graph represent individual query operations to perform.
+The edges of the graph represent the flow of data from one operation to another.
+In a Flux query plan, data flows vertically from the leaves of the DAG to the roots.
```

Comment:
It would be good to mention that we think of the direction of data flow as going upwards from sources to terminal yield nodes.  This helps contextualize terms like "push down" and "pull up".